Customizing the gem to provision user to select variant

kanwal0801 opened the pull request:
User can now select variant of assembly product on cart page.

Commit:
Customizing the gem to provision user to select variant

Commit:
Checking in missing migration

BuckBuchanan commented:
@kanwal0801 Where in the code are you creating entries in the assemby_variants table? I can't find it.

It looks like you started to do something in the app/controllers/spree/orders_controller_decorator.rb inside the gobigwin app. But didn't finish implementing it. Maybe?

I don't see where you ever grab the "part[<%= part.id %>]" fields from the form and map them to assembly_variants. So line_item.assembly_variants is always going to be an empty array. Therefore, no inventory units / adjustments to count-on-hand.

Commit:
Ability to check whether the assembly parts variants are in stock or not

Commit:
Ability to add assembly product multiple times by choosing
different set of variants
Displaying variants info on cart page, admin order details
page
